packages/core/src/lib/waku_store/index.ts

```diff
@@ -96,7 +90,7 @@ export interface QueryOptions {
    * The cursor index will be exclusive (i.e. the message at the cursor index will not be included in the result).
    * If undefined, the query will start from the beginning or end of the history, depending on the page direction.
    */
-  cursor?: Cursor;
+  cursor?: Index;
```

Comment:
Rename the type to `Cursor` so that variable and type have the same name.

Reply:
The name `Index` comes from the `proto` definitions; changing it will lead to inconsistencies.